docs/ref/django-admin.txt

```diff
@@ -1559,6 +1559,10 @@ Creates a superuser account (a user who has all permissions). This is
 useful if you need to create an initial superuser account or if you need to
 programmatically generate superuser accounts for your site(s).
 
+.. django-admin-option:: --noinput, --no-input
+
+Runs the command in `non-interactive` mode and suppresses all user prompts.
```

Comment:
This should moved to options (before `--username`). Maybe sth similar to the description for `makemessages`, e.g.
```
Suppresses all user prompts. If a suppressed prompt cannot be resolved automatically, the command will exit with error code 1.
```